projects/components/src/table/table.component.scss

```diff
@@ -40,7 +40,6 @@ $header-height: 32px;
 .data-row {
   display: flex;
   flex-direction: row;
-  height: 44px;
```

Comment:
Looks fine to me. @jake-bassett  thoughts?

Reply:
This change still breaks one table. Need to fix that case first.

Reply:
@itssharmasandeep Could you please look into this issue so that we can go ahead and merge this?